pkgs/development/python-modules/chime/default.nix

```diff
@@ -0,0 +1,32 @@
+{
+  alsa-utils,
+  buildPythonPackage,
+  fetchFromGitHub,
+  lib,
+  poetry-core,
+  stdenv,
+}:
+
+buildPythonPackage {
+  pname = "chime";
+  version = "unstable-2023-03-06";
```

Comment:
I am not sure if the version is exactly what it should be. I'd suggest you start packaging one version before the last, do all the stuff then use that `maintainers/scripts/update.nix` to update it to the latest version, that would define the right version.